src/script/conversation/ClientMismatchHandler.ts

```diff
@@ -102,7 +101,7 @@ export class ClientMismatchHandler {
       const unknownUsers = getDifference(
         knownUsers.map(user => user.id),
         missingUserIds,
-      ).map(id => ({domain: Config.getConfig().FEATURE.FEDERATION_DOMAIN || '', id}));
+      ).map(id => ({domain: '', id}));
```

Comment:
The mismatch handler is only called in a non-federated env. This whole code will be retired when we use the `core` for sending messages (which is the case for federated messages)